New html term label (#1268)
Codecademy/docs: content/html/concepts/elements/terms/label/label.md

---
Title: '<label>'
Description: 'Identifies captions for HTML Form elements such as <input> and other UI elements.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Elements'
  - 'Form'
  - 'Label'
  - 'Input'
  - 'Interface'
  - 'Tags'
  - 'UI'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<label/>` element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [**`<input>`**](https://www.codecademy.com/resources/docs/html/elements/input/input.md) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.

Reviewer: ```suggestion
The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input/input.md) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
```

## Syntax

```html
<label>Input Label</label>
```

## Usages

The `<label>` tag can define a label for several elements:

- [`<input>`](../input/input.md)
- [`<meter>`](../meter/meter.md)
- [`<progress>`](../progressput/progress.md)
- [`<select>`](../select/select.md)
- [`<textarea>`](../textarea/textarea.md)

> ```suggestion

Reviewer: You still need to remove ```suggestion blocks around text.

> **Note:** `<label>` tags are essential for good accessbility.
```

> ```suggestion

Reviewer: Remove ```suggestion blocks around text

> - '<label>' tags creates captions for its coresponding UI elements.
```

> ```suggestion

Reviewer: Remove ```suggestion blocks around text.

---
Title: '<label>'
Description: 'Identifies captions for HTML Form elements such as <input> and other UI elements.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Elements'
  - 'Form'
  - 'Label'
  - 'Interface'
  - 'Tags'
  - 'UI'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [**`<input>`**](../input/input.md) element. The <label> tag also supports the *Event Attributes* and the *Global Attributes* in HTML.

Reviewer: ```suggestion
The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
```

## Syntax

```html
<label>Input Label</label>
```

## Usages

> ```suggestion

Reviewer: Remove the suggestion block, just leave the sentence.

Reviewer: Still needs to be done

> The <label> tag can define a label for several elements:

Reviewer: ```suggestion
The `<label>` tag can define a label for several elements:
```

```

> - [**`<input>`**](../input/input.md)

Reviewer: Remove the bold on these.

- [**`<meter>`**](../meter/meter.md)
- [**`<progress>`**](../progressput/progress.md)
- [**`<select>`**](../select/select.md)
- [**`<textarea>`**](../textarea/textarea.md)

```suggestion
> **Note:** `<label>` tags are essential for good accessbility.
>

> > - They allow screenreaders to connect this caption to its coresponding ui elements.

Reviewer: ```suggestion
> - They allow screen readers to connect this caption to its corresponding UI elements.
```

> > - Also, when a user clicks the text within the **`<label>`** element, it toggles the coresponding input. This is great for users who have difficulty clicking on very small ui elements such as checkboxes.

Reviewer: ```suggestion
> - Also, when a user clicks the text within the `<label>` element, it toggles the corresponding input. This is great for users who have difficulty clicking on very small UI elements such as checkboxes.
```

```

## Atributes

Some common attributes used to set properties for the `<label>` element are shown below:

|   Attribute    |   Data Type   | Description
|   `for`    |   String   | Specifies the id of the form element the `<label>` should be bound to.
|   `form`    |   String   | Specifies which form the `<label>` belongs to.

> ```suggestion

Reviewer: remove the suggestion block and just leave the sentence.

Multiple label elements can be used for the same UI element.
```

```html
<form id="contact">
  <label form="contact" for="name">Input Label</label>
  <input id="name" />
</form>
```

> ```suggestion

Reviewer: Remove the suggestion block and just leave the sentence.

> **Note:** UI elements can be connected to any **`<label>`** element when placed between `<label></label>` tags.
```

```suggestion
The following examples uses a `<form>` element that features `<input>` elements with corresponding `<label>` tags:
```

```html
<label for="name">
  Name Input Label
  <input id="name" />
</label>
```

## Multiple labels

Multiple `<label>` elements can be used for the same ui element.

```html
<label for="name">Name Input Label
<input id="name" />
<label for="name">Warning: Name is required</label>
```

> ```suggestion

Reviewer: Remove the suggestion block and just leave the sentence.

> **Note:** Do not place anchors, buttons, or any interactive UI element, between  `<label></label>` tags.
```

## Example

> ```suggestion

Reviewer: Remove the suggestion block and just leave the sentence.

> # Label

Reviewer: The metadata for this entry is missing. You can look at the source for one of the other HTML entries to see what it looks like.  Also review the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md).

The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [**`<input>`**](../input/input.md) element. The <label> tag also supports the *Event Attributes* and the *Global Attributes* in HTML.

## Syntax

```html
<label>Input Label</label>
```

## Usages

> The <label> tag defines a label for several elements:

Reviewer: ```suggestion
The <label> tag can define a label for several elements:
```

- [**`<input>`**](../input/input.md)
- [**`<meter>`**](../meter/meter.md)
- [**`<progress>`**](../progressput/progress.md)
- [**`<select>`**](../select/select.md)
- [**`<textarea>`**](../textarea/textarea.md)

> **Note:** Labels are important for accessbility. They allow screenreaders to connect this caption to its coresponding ui elements.

Reviewer: ```suggestion
> **Note:** Labels are important for accessibility. They allow screenreaders to connect this caption to its corresponding UI elements.
```

> Also, when a user clicks the text within the **`<label>`** element, it toggles the coresponding input. This is great for users who have difficulty clicking on very small ui elements such as checkboxes.

Reviewer: ```suggestion
Also, when a user clicks the text within the `<label>` element, it toggles the corresponding input. This is great for users who have difficulty clicking on very small UI elements such as checkboxes.
```

## Atributes

Some common attributes used to set properties for the `<label>` element are shown below:

|   Attribute    |   Data Type   | Description
|   `for`    |   String   | Specifies the id of the form element the label should be bound to.
|   `form`    |   String   | Specifies which form the label belongs to.

```html
<form id="contact">
  <label form="contact" for="name">Input Label</label>
  <input id="name" />
</form>
```

> **Note:** UI elements can be connected to any **`<label>`** element when placed between **`<label></label>`** tags.

Reviewer: ```suggestion
> **Note:** UI elements can be connected to any `<label>` element when placed between `<label></label>` tags.
```

```html
<label for="name">
  Name Input Label
  <input id="name" />
</label>
```

## Multiple labels

> Multiple label elements can be used for the same ui element.

Reviewer: ```suggestion
Multiple label elements can be used for the same UI element.
```

```html
<label for="name">Name Input Label
<input id="name" />
<label for="name">Warning: Name is required</label>
```

> **warning:** Do not place anchors, buttons, or any interactive ui element, between  **`<label></label>`** tags.

Reviewer: ```suggestion
> **Note:** Do not place anchors, buttons, or any interactive UI element, between  `<label></label>` tags.
```

## Example

> The following examples uses a `<form>` element that features `<input>` elements with coresponding `<label>` tags:

Reviewer: ```suggestion
The following examples uses a `<form>` element that features `<input>` elements with corresponding `<label>` tags:
```

Reviewer: This still needs to be done.
